autotests/cross_signing_data.json

```diff
@@ -0,0 +1,926 @@
+{
+  "device_keys": {
```

Comment:
Do you need that many keys in the test data?

Also - I assume that `@aloy:kde.org`, Carl and you wilfully exposed your data, including device display names? This is not something normally shown to other users.

Reply:
Right, i removed most devices. I randomized the display names a bit. For the rest, i'm personally fine with it, since it's not secret data. I'll ping @CarlSchwan  whether he's fine with it. `@aloy:kde.org` is my test account :)

Reply:
carl mentioned he's fine with it